### DISCUSSION TITLE
Errors of running basic examples

### DISCUSSION THREAD

**Discussion by hugolin615** [GMLC-TDC/HELICS#2386] [General]
I encountered some errors while running the basic example.
$ helics run --path=fundamental_default_runner.json
helics-cli [info]: Running federation: fundamental_default
helics-cli [info]: Running federate Charger as a background process
helics-cli [info]: Running federate Battery as a background process
helics-cli [info]: Waiting for 3 processes to finish ...
Error: Process Charger has failed, killing other processes
helics-cli [error]: Process broker exited with return code -9
helics-cli [error]: Process Charger exited with return code 1
helics-cli [error]: Process Battery exited with return code 1
helics-cli [info]: Done.
Exception in thread Thread-1:
Traceback (most recent call last):
File "/usr/lib/python3.10/threading.py", line 1009, in _bootstrap_inner
self.run()
File "/home/hugo/.local/lib/python3.10/site-packages/helics_cli/status_checker.py", line 40, in run
raise HELICSRuntimeError("Error has occurred")
helics_cli.exceptions.HELICSRuntimeError: Error has occurred
Any helps? OS is Ubuntu 22.04 and python version is 3.10.4. I installed helics and helics-cli by using pip.

**Comment by trevorhardy** [comment]
As mentioned on Gitter, I've tested the examples using HELICS v3.2.1 and Python v3.10.4 on both my Mac and on a clean Ubuntu 22.04 VM (both using Anaconda virtual environments) and was able to get the fundamental default example to run. Look through the log files (broker.log, Charger.log, and Battery.log) to see if there are any more clues on what is causing the error.

**Comment by hugolin615** [comment]
As I replied in Gitter, I pip uninstall helics and helics-cli and pip install them again in a conda environment. It is working now. Before I do that, I can see that previous broker.log complains that zmq connections cannot be established. Maybe the port is occupied? I am not sure, as I don't remember that I have other process using this protocol.  
Thanks again!
